Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby "2.3.0"
```

Comment:
Yeah, like i said, the correct way to define this isn't exactly clear, and its a dev dependency. Since ruby (should) have appropriate BC, it seems best to suggest the minimum required version for all features, hence 2.2+